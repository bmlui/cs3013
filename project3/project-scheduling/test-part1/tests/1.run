./scheduler 0 FIFO tests/1.in 0
