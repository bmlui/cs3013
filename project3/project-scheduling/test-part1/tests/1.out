Execution trace with FIFO:
t=0: [Job 0] arrived at [0], ran for: [3]
End of execution with FIFO.
